cmd/kubectl-kubestellar/cmd/ensure/wds.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Sub-command for ensuring the existence of a workload description space (WDS),
+// along with requisite APIBindings.
+// The WDS name is given as a required command-line argument.
+// --with-kube is a required flag which determines if root:compute APIBindings are needed.
+
+package ensure
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/klog/v2"
+
+	kcpclientset "github.com/kcp-dev/kcp/pkg/client/clientset/versioned"
+
+	plugin "github.com/kubestellar/kubestellar/pkg/cliplugins/kubestellar/ensure"
+)
+
+var withKube bool // Variable for --with-kube flag
+
+// Create the Cobra sub-command for 'kubectl kubestellar ensure wds'
+func newCmdEnsureWds(cliOpts *genericclioptions.ConfigFlags) *cobra.Command {
+	// Make wds command
+	cmdWds := &cobra.Command{
+		Use:     "wds <WDS_NAME> --with-kube=<TRUE/FALSE>",
+		Aliases: []string{"wmw"},
+		Short:   "Ensure existence and configuration of a workload description space (WDS, formerly WMW)",
+		Args:    cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// At this point set silence usage to true, so that any errors
+			// following do not result in the help being printed. We only
+			// want the help to be displayed when the error is due to an
+			// invalid command.
+			cmd.SilenceUsage = true
+			err := ensureWds(cmd, args, cliOpts)
+			return err
+		},
+	}
+
+	// Add flag for
+	cmdWds.Flags().BoolVar(&withKube, "with-kube", true, "Include root:compute API bindings")
```

Comment:
I think it would be clearer if the help string spoke of "Kubernetes API bindings" rather than "root:compute API bindings".